source/ui/Components/Output/PixelDisplay.qml

```diff
@@ -0,0 +1,221 @@
+import QtQuick 2.0
+import QtQuick.Window 2.0
+import QtQuick.Controls 1.4
+import eragpsim.pixeldisplaypainteditem 1.0
+
+Item {
+    property var outputItemIndex: 3
+
+    /*Rectangle {//Dummy-Module, will be replaced
+        id: greenBackground
+        width: 360
+        height: 360
+        // color: "green"
+        anchors.fill: parent
+    }*/
+    /*Image {
+        //width: 100
+        //height: 50
+        smooth: false
+        id: pixelDisplayImage
+        source: "image://pixeldisplayprovider/red"
+        anchors.fill: parent
+    }*/
+
+    // Update the output item's content (there may already be some initial
+    // values in memory).
+    Component.onCompleted: {
+    }
+    PixelDisplayPaintedItem{
+        id: pixeldisplaypainteditemid
+        anchors.fill: parent
+        outputComponentPointer : outputComponent
+    }
+
+    // Connect the output item to signals that the model might send.
+    Connections {
+        target: outputComponent
+        onNewImage: pixeldisplaypainteditemid.setImage(image)
+        // Send when the memory changes (at any address).
+        onMemoryChanged: {
+            console.log("onMemoryChanged");
```

Comment:
Please remove console (debug) output in tested code which is ready to be merged on master.